### ISSUE TITLE
Choice of LinkConstraint name

### ISSUE THREAD

**Issue by pgkirsch** [hoburg/gpkit#587]
Brace yourself, this is going to sound unbelievably picky: The name LinkConstraint seems a little odd to me for a couple reasons.

Firstly, it sounds like a method rather than a class (as in you are linking a constraint).

Secondly, seeing as it inherits from a ConstraintSet and comprises many constraints, the fact it is a singular "Constraint" doesn't sound right.

Suggestions for possible alternatives:
- LinkedConstraintSet (does what it says on the tin, but probably too long)

... that's all I have at the moment, but open to other suggestions.

EDIT: the standard syntax could be `import LinkedConstraintSet as LCS` if that helps at all

**Comment by bqpd** [comment]
Ha, I don't like it either.

LinkedSet?

**Comment by bqpd** [comment]
Oh, I forgot in vtail, but instead of 

```python
        wb = WingBox()
        wb.subinplace({"taper": taper})
        lc = LinkConstraint([constraints, wb])
```

you could do 

```python
        wb = WingBox()
        wb.subinplace({"taper": taper})
        [...]
        Model.__init__(self, objective, wb.link(constraints), **kwargs)
```

**Comment by whoburg** [comment]
I'm liking the idea that the `wb.link` returns a `LinkedConstraintSet`.

**Comment by bqpd** [comment]
Sounds good to me. @pgkirsch, you good with that name?

**Comment by pgkirsch** [comment]
Yup!

**Comment by bqpd** [comment]
Fixed in #605
